File: jclang/src/main/java/jclang/structs/CXString.java


package jclang.structs;

import com.sun.jna.Pointer;
import com.sun.jna.Structure;

@SuppressWarnings("unused")
@Structure.FieldOrder({"data", "privateFlags"})
public class CXString extends Structure {
    public Pointer data;
    public int privateFlags;

    public static class ByValue extends CXString implements Structure.ByValue {}
}

File: klang/libclang/src/main/kotlin/klang/jvm/binding/CXString.kt
package klang.jvm.binding

import com.sun.jna.Pointer
import com.sun.jna.Structure

@Suppress("unused")
@Structure.FieldOrder("data", "privateFlags")
class CXString : Structure() {
    @JvmField var data: Pointer? = null
    @JvmField var privateFlags = 0
}